### PR TITLE
[Feature] : Ourservice component

### PR DESCRIPTION
**Pull request by Himanshushakyaa**
## Description

-   [Ticket Link](#) (if any)
- create ourservice components with fully responsive and animated

## UI screenshot or Video

https://github.com/primathonacademy/april-batch-live-project/assets/131392846/8191cd4e-e1aa-4249-ac6e-2a3876975fd1

-  

---

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [ ] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [ ] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/assets/images/index.js
Comment (ram-primathon) [inline src/assets/images/index.js:1]: move this index file inside assets folder

File: src/assets/images/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { default as ourServiceImage1} from "/src/assets/images/our-service-1.png"
```
Comment (ram-primathon) [inline src/assets/images/index.js]: make it a relative import

File: src/assets/images/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+export { default as ourServiceImage2} from "/src/assets/images/our-service-2.png"
```
Comment (ram-primathon) [inline src/assets/images/index.js]:
```suggestion
export { default as ourServiceImage2} from "./homepage/our-service-2.png"
```

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/assets/images/our-service-1.png
Comment (ram-primathon) [inline src/assets/images/our-service-1.png:1]: move this to homepage folder

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/assets/images/our-service-1.png
Comment (Himanshushakyaa) [reply src/assets/images/our-service-1.png:1]: sir iye smjh nhi ayaa

**Review by ram-primathon — APPROVED** [review]

**Comment by ram-primathon** [comment]
Good Job 👍. Thank you.

**Pull request merged**